draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -366,7 +373,9 @@ public boolean isClosed() {
     if (activePointerId == INVALID_POINTER) {
       return false;
     }
-    viewDragHelper.processTouchEvent(ev);
+    if(getContext().getResources().getConfiguration().orientation == Configuration.ORIENTATION_PORTRAIT){
+      viewDragHelper.processTouchEvent(ev);
+    }
```

Comment:
@pedrovgs I thought it is better to give all the options (all, landscape, portrait) for the user, in case of different needs I'll be adding a flag to support all of those to trigger touch events. You can check it out after the commit with new implementations.